### ISSUE TITLE
Add `CITATION.cff` file here?

### ISSUE THREAD

**Issue by BenjaminRodenberg** [precice/precice#1703]
**Please describe the problem you are trying to solve.**

Better visibility and usability, if I want to cite preCICE (from the perspective of somebody accessing the github repository, not the website)

**Describe the solution you propose.**

There is the possibility to provide a `CITATION.cff` file for providing citation information. See https://github.blog/2021-08-19-enhanced-support-citations-github/

As far as I understand CFF we should point to the distribution, because this is the closest thing that we do which is closest to publishing on zotero. We would then point to the paper as a `preferred-citation`. (similar for the FEniCS-preCICE adapter, see https://github.com/precice/fenics-adapter/pull/157)

**Describe alternatives you've considered**

We already have information about citation on our website.

**Comment by uekerman** [comment]
Could we (also?) link from the citation file to our [citation guide](https://precice.org/fundamentals-literature-guide.html)?

**Comment by BenjaminRodenberg** [comment]
> Could we (also?) link from the citation file to our [citation guide](https://precice.org/fundamentals-literature-guide.html)?

I guess so. There is the following

```
message: "If you use this software, please cite it using these metadata."
```

Which can be changed arbitrarily. It is then rendered as follows:

![Screenshot from 2023-07-05 20-59-58](https://github.com/precice/precice/assets/5740604/dca8cd80-ad7a-468f-94cd-2d22dee567ba)

See https://github.com/precice/fenics-adapter/pull/157 for some inspiration.

**Comment by fsimonis** [comment]
I can vaguely remember that we discussed this when it was released and decided not to go with it, as the citation guide is the way to go.

We should definitely reconsider if things have changed.

**Comment by uekerman** [comment]
Yes, I also remember we discussed this already. Can we make it in a way such that there is no paper / DOI / etc, but only the link to the citation guide?
